pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -378,65 +394,74 @@ func (m *manager) removeStaleState() {
 	assignments := m.state.GetCPUAssignments()
 	for podUID := range assignments {
 		for containerName := range assignments[podUID] {
+			lh := klog.LoggerWithValues(lh_, "podUID", podUID, "containerName", containerName)
+
 			if _, ok := activeContainers[podUID][containerName]; ok {
-				klog.V(5).InfoS("RemoveStaleState: container still active", "podUID", podUID, "containerName", containerName)
+				lh.V(5).Info("container still active")
 				continue
 			}
-			klog.V(2).InfoS("RemoveStaleState: removing container", "podUID", podUID, "containerName", containerName)
-			err := m.policyRemoveContainerByRef(podUID, containerName)
+
+			lh.V(2).Info("removing container")
+			err := m.policyRemoveContainerByRef(ctx, podUID, containerName)
 			if err != nil {
-				klog.ErrorS(err, "RemoveStaleState: failed to remove container", "podUID", podUID, "containerName", containerName)
+				lh.Error(err, "failed to remove container")
 			}
 		}
 	}
 
 	m.containerMap.Visit(func(podUID, containerName, containerID string) {
+		lh := klog.LoggerWithValues(lh_, "podUID", podUID, "containerName", containerName)
 		if _, ok := activeContainers[podUID][containerName]; ok {
-			klog.V(5).InfoS("RemoveStaleState: containerMap: container still active", "podUID", podUID, "containerName", containerName)
+			lh.V(5).Info("containerMap: container still active")
 			return
 		}
-		klog.V(2).InfoS("RemoveStaleState: containerMap: removing container", "podUID", podUID, "containerName", containerName)
-		err := m.policyRemoveContainerByRef(podUID, containerName)
+		lh.V(2).Info("containerMap: removing container")
+		err := m.policyRemoveContainerByRef(ctx, podUID, containerName)
```

Comment:
Perhaps this could be a nice place to toy around with LoggerV and have log entries based on log verbosity (https://kubernetes.slack.com/archives/C20HH14P7/p1737563609490089)?